preloadResponse type declaration conflicts with typescript webworker declaration

**Library Affected**:
workbox-strategies v6.4.0

(6.3.0 was working)

**Browser & Platform**:
all browsers, Typescript 4.2.3

**Issue or Feature Request Description**:

On typescript build I get this error:

> ../../node_modules/workbox-strategies/index.d.ts:11:18 - error TS2717: Subsequent property declarations must have the same type.  Property 'preloadResponse' must be of type 'Promise<any>', but here has type 'Promise<Response>'.
> 
> 11         readonly preloadResponse: Promise<Response | undefined>;
>                     ~~~~~~~~~~~~~~~
> 
>   ../../node_modules/typescript/lib/lib.webworker.d.ts:1572:14
>     1572     readonly preloadResponse: Promise<any>;
>                       ~~~~~~~~~~~~~~~
>     'preloadResponse' was also declared here.

Typescript has this interface declaration:

```typescript
interface FetchEvent extends ExtendableEvent {
    readonly clientId: string;
    readonly preloadResponse: Promise<any>;
    readonly replacesClientId: string;
    readonly request: Request;
    readonly resultingClientId: string;
    respondWith(r: Response | Promise<Response>): void;
}
```

Which conflicts with index.d.ts in workbox-strategies:

```typescript
declare global {
    interface FetchEvent {
        readonly preloadResponse: Promise<Response | undefined>;
    }
}
```

--- a/packages/workbox-strategies/src/StrategyHandler.ts
+++ b/packages/workbox-strategies/src/StrategyHandler.ts
@@ -155,7 +155,9 @@ class StrategyHandler {
       event instanceof FetchEvent &&
       event.preloadResponse
     ) {
-      const possiblePreloadResponse = await event.preloadResponse;
+      const possiblePreloadResponse = (await event.preloadResponse) as
+        | Response
+        | undefined;
       if (possiblePreloadResponse) {
         if (process.env.NODE_ENV !== 'production') {
           logger.log(

--- a/packages/workbox-strategies/src/index.ts
+++ b/packages/workbox-strategies/src/index.ts
@@ -18,7 +18,8 @@ import './_version.js';
 // See https://github.com/GoogleChrome/workbox/issues/2946
 declare global {
   interface FetchEvent {
-    readonly preloadResponse: Promise<Response | undefined>;
+    // See https://github.com/GoogleChrome/workbox/issues/2974
+    readonly preloadResponse: Promise<any>;
   }
 }
 

